src/computer.cpp

```diff
@@ -47,13 +100,28 @@ computer &computer::operator=(const computer &rhs)
     security = rhs.security;
     name = rhs.name;
     mission_id = rhs.mission_id;
-    options = rhs.options;
-    failures = rhs.failures;
+    options.resize(rhs.options.size());
```

Comment:
I've been trying to debug an issue with options showing up in from-JSON buildings that lose all of their info somewhere in the process, and this was one of the attempts at tracking down that issue.  Also, it's still there so if you see anything glaringly wrong, let me know.